### PR TITLE
Added VDSR

### PR DESCRIPTION
**Pull request by ayush12gupta**

### CODE REVIEW

File: super_resolution/VDSR_PyTorch/README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+## Implementation
+
+We first preprocess the dataset 291I so as to produce data argumented images by ratating, flipping and producing images of different scales(x2 , x3, x4) so as to support multi scale SR, and produce images of 41x41 dimension. 
+We train our model using  MSE Loss for 50 epochs and appling gradient clipping and evaluate out model on some images extraced from Set5.
+
+## Result
+
+**Ground Truth Image**
+
+![gt](/assets/GT.png)
```
Comment (nirmal-suthar) [inline super_resolution/VDSR_PyTorch/README.md]:
```suggestion
![gt](./assets/GT.png)
```

File: super_resolution/VDSR_PyTorch/README.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+**4x scaled Input Image**
+
+![input](/assets/Input.png)
```
Comment (nirmal-suthar) [inline super_resolution/VDSR_PyTorch/README.md]:
```suggestion
![input](./assets/Input.png)
```

File: super_resolution/VDSR_PyTorch/README.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+**Super Resolution Output Image**
+
+![out](/assets/Output.png)
```
Comment (nirmal-suthar) [inline super_resolution/VDSR_PyTorch/README.md]:
```suggestion
![out](./assets/Output.png)
```

**Comment by nirmal-suthar** [comment]
Remove unused files from ./assets/Set5 (there are some files which are not used)

**Comment by ayush12gupta** [comment]
Done

File: super_resolution/VDSR_PyTorch/main.py [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+            optimizer.step()
+
+            if i % 100 == 0:
+                print("Epoch[{}]({}/{}): Loss: {:.10f}".format(epoch, i, len(dataloader), loss.data[0]))
+
+        if epoch % 2 == 0:
+            save_checkpoint(model, epoch)
+
+def eval( model, args):
+    im_gt = Image.open(args.test_data+"GT.bmp").convert("RGB")
```
Comment (nirmal-suthar) [inline super_resolution/VDSR_PyTorch/main.py]: This should be  GT.png right?

File: super_resolution/VDSR_PyTorch/main.py [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    im_b = Image.open(args.test_data+"Input.bmp").convert("RGB")
```
Comment (nirmal-suthar) [inline super_resolution/VDSR_PyTorch/main.py]: Same here

**Review by nirmal-suthar — APPROVED** [review]

File: super_resolution/VDSR_PyTorch/main.py (hunk 5 above)
Comment (ayush12gupta) [reply super_resolution/VDSR_PyTorch/main.py]: Done

**Comment by ayush12gupta** [comment]
Done

**Pull request merged**